scipy/stats/_continuous_distns.py

```diff
@@ -9229,6 +9229,41 @@ def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
         return super().expect(func, args, loc,
                               scale, lb, ub, conditional, **kwds)
 
+    @_call_super_mom
+    @extend_notes_in_docstring(rv_continuous, notes="""\
+        The `scale` parameter is ignored.\n\n""")
+    def fit(self, data, *args, **kwds):
+        if kwds.pop('superfit', False):
+            return super().fit(data, *args, **kwds)
+
+        data, fshape, floc, fscale = _check_fit_input_parameters(self, data,
+                                                                 args, kwds)
+
+        def find_mu(data):
+            return stats.circmean(data)
+
+        def find_kappa(data):
+            r = 1 - stats.circvar(data)
+
+            def solve_for_kappa(kappa):
+                return sc.i1e(kappa)/sc.i0e(kappa) - r
+
+            root_res = root_scalar(solve_for_kappa, method="brentq",
+                                   bracket=(1e-8, 1e9))
+            if root_res.converged:
+                kappa = root_res.root
+                return kappa
+            else:
+                raise FitSolverError("Shape parameter could not be estimated.")
```

Comment:
This need to be tested.

Reply:
Could not come up with an example where this failed, so removed it.

Other Error message still come to mind:
* the `data` must strictly lie in the interval [0, 2*np.pi], otherwise the circular mean will not make sense. Is sth like a DomainError in the distribution fitting machinery?